### ISSUE TITLE
Questions about Dataset and Experiments

### ISSUE THREAD

**Issue by LLMforScience** [RLHFlow/Online-DPO-R1#8]
Thank you for your work.

I noticed that the MATH dataset you used contains 7.5K questions. However, the Qwen2.5-7B-SimpleRL model uses 8,523 questions. Could you please clarify the differences?

**Comment by WeiXiongUST** [comment]
MATH = training set 7.5K + test set 5 K. However, in practice, researchers often only use a fixed subset consisting of 500 samples in the test set to measure the mathematical reasoning ability of LLM. So SimpleRL project uses 7.5K + 4.5K (from test set) to train the model. They also filter out the easy prompt so they have 8523 eventually.

**Comment by WeiXiongUST** [comment]
Hi, we use this dataset: RLHFlow/qwq_gen_sft_15k for the SFT training. The config is 

https://wandb.ai/axolotl-ai/qwen-im-end/runs/uz5v11h9/files/tmp/axolotl_config_g2g7wwya.yml

but we use a learning rate of 1e-5, a global batch size of 32, and train for 1 epoch.

**Comment by WeiXiongUST** [comment]
Yes, there are multiple responses per prompt.

**Comment by WeiXiongUST** [comment]
Actually we do not generate the data by ourselves but use a randomly selected subset of  justus27/qwq_cot_sampled_math
